dev/tests/api-functional/testsuite/Magento/GraphQl/Review/GetProductReviewsTest.php

```diff
@@ -204,7 +281,7 @@ public function testProductReviewRatings()
      */
     public function testCustomerReviewsAddedToProduct()
     {
-        $query = <<<QUERY
+        $query             = <<<QUERY
```

Comment:
Hi @huykon. Could I ask you to double-check the indentation for all your changes, please? I'm not sure what happened but for some reason, we have unnecessary spaces here and there. 

Thanks.